client/src/components/VersionInfo.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <p>
+      Version info:
+    </p>
+    <p v-for="component in versioninfo" :key="component.name">{{component.name}}
+        {{component.version}}</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+const API_URL = 'http://localhost:4000';
```

Comment:
I'd suggest pulling out config values like this into a an object in a separate file. Deployment is some way off, but probably good to separate out values like this which may be overwritten when deploying.